README.md

```diff
@@ -172,6 +176,4 @@ If you are reading this it looks like you are looking to add an egg to your serv
 
 [Vintage Story](/vintage_story/vintage_story/)
 
-[Xonotic](/xonotic/xonotic/)
-
-[Cryofall](/cryofall/)
```

Comment:
I don't care what you do generally but try and keep the edits to the game and not touch other things...